src/output/output.cpp

```diff
@@ -269,8 +264,21 @@ bool wf::output_impl_t::activate_plugin(const plugin_grab_interface_uptr& owner,
             return false;
     }
 
+    return true;
+}
+
+bool wf::output_impl_t::activate_plugin(const plugin_grab_interface_uptr& owner,
+    bool ignore_inhibit)
+{
+    if (!can_activate_plugin(owner) || !ignore_inhibit)
```

Comment:
Oh hm I didn't see this until now. `ignore_inhibit` should also be a parameter of `can_activate_plugin` (as I said, we do not want any differences between the two functions in terms of when one succeeds or fails.

Reply:
It seems to me that it is pointless to pass an argument that just makes the function fail. Why would you not handle this in the plugin(s) and just avoid calling (can_)activate in the first place?

Reply:
Fixed.